output/plugins/additional/xml/additional.xml

```diff
@@ -81,16 +81,23 @@ Written by
     <property name="tab_width" type="text" help="The number of spaces per tab character.">4</property>
   </objectinfo>
 
-  <objectinfo class="wxToggleButton" startgroup="1" icon="toggle_button.xpm" type="widget">
+
+  <objectinfo class="wxToggleButton" icon="toggle_button.xpm" type="widget">
+    <inherits class="wxAnyButton"/>
     <inherits class="wxWindow"/>
     <inherits class="AUI"/>
     <inherits class="Validator"/>
     <property name="name" type="text">m_toggleBtn</property>
-    <property name="label" type="wxString_i18n">Toggle me!</property>
     <property name="value" type="bool">0</property>
     <event name="OnToggleButton" class="wxCommandEvent" help="Handles a wxEVT_TOGGLEBUTTON event."/>
   </objectinfo>
 
+  <objectinfo class="wxBitmapToggleButton" icon="bitmap_toggle_button.xpm" type="widget">
```

Comment:
The specified icon file does not exist, i think you had one before, maybe it got lost during force-push?